### ISSUE TITLE
How to find the dropout layer in googleNet-dag.net

### ISSUE THREAD

**Issue by wendygo** [vlfeat/matconvnet#368]
Hi~ Thank you very much for so cool works, which give me great help.
The GoogLeNet incarnation of the Inception architecture in "Going Deeper with Convolutions" shows that dropout layer is the third last layer. the last four layers are pool(1*1*1024), dropout(1*1*1024),linear(1*1*1000),softmax(1*1*1000). In the pre-trained googleNet offered by matconvnet, the last four layers are icp9_out(7*7*1024),cls3_pool(1*1*1024),cls3_fc(1*1*1000),prob(1*1*1000). Whether cls3_fc corresponding to linear and cls_pool corresponding to pool? Then where is the dropout layer?

I saw the functions in matconvnet shows the explanation about the "mode" as follows:
This flag can either be normal or test. In the latter case, certain blocks switch to a test mode suitable for validation or evaluation as opposed to training. For instance, dropout becomes a pass-through block in test mode. 
When I change the mode from normal to test, I also cannot find the dropout layer.

Looking forward to replay.
Thank you very much in advance and Happy New Year!

**Comment by lenck** [comment]
Hi, the GoogLeNet in MatConvNet is imported without the DropOut layers... So, the change of mode does not have any influence...
